src/main/resources/data/minecolonies/tags/items/cook_ingredient.json

```diff
@@ -7,6 +7,8 @@
       "minecraft:chicken",
       "minecraft:porkchop",
       "minecraft:rabbit",
-      "minecraft:potato"
+      "minecraft:potato",
+      "minecraft:kelp",
+      "minecraft:dried_kelp"
```

Comment:
did you want minecraft:dried_kelp_block here too, so you can uncompress them?

Reply:
Yes I did, thanks.